src/controllers/apartment.ts

```diff
@@ -124,8 +124,8 @@ const validateLink = (link: string) => {
  * This does the actual searching for apartments in the database
  */
 export const postSearchForApartments = async (req: Request, res: Response, next: NextFunction) => {
-    await check("numBedrooms", "Number of bedrooms must be a number (without commas).").exists().isNumeric().run(req);
-    await check("numBathrooms", "Number of bathrooms must be a number (without commas).").exists().isNumeric().run(req);
+    await check("numBedrooms", "Number of bedrooms must be a number (without commas).").exists().custom((value: string) => validatePrice(value)).run(req);
+    await check("numBathrooms", "Number of bathrooms must be a number (without commas).").exists().custom((value: string) => validatePrice(value)).run(req);
```

Comment:
It doesn't make sense for the number of bathrooms in a condo to be a price. Like it shouldn't allow people to say numBathrooms = $1,200.